### ISSUE TITLE
Feature request: RSS feeds

### ISSUE THREAD

**Issue by sirMackk** [GriffinMB/keyserve#1]
A blog with an RSS feed is much easier to keep track of.

**Comment by GriffinMB** [comment]
Hmm. It's a fairly dumb server at the moment. Maybe I could make it so you could specify a directory (or directories), and anything in that directory will be automatically added to a simple RSS feed? 

I should probably go ahead and add a config file to the process since options are starting to grow.

**Comment by sirMackk** [comment]
Config file sounds good. What do you think about generating a feed from the
files in the blog directory? I could probably make a PR for this :D

On Apr 3, 2017 12:52, "Griffin Byatt" <notifications@github.com> wrote:

Hmm. It's a fairly dumb server at the moment? Maybe I could make it so you
could specify a directory (or directories), and anything in that directory
will be automatically added to a dumb RSS feed?

I should probably go ahead and add a config file to the process since
options are starting to grow.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/GriffinMB/keyserve/issues/1#issuecomment-291203511>,
or mute
the thread

**Comment by GriffinMB** [comment]
The only problem is that it's intended to be pretty free-form. All of my posts are in a "posts" directory, and there's some nonsense in the root "blog" directory -- like the `reset.txt` file, the index page, etc. 

We could do it so that by default it generates a feed from `.md` files in the blog directory? That way it ignores all the subdirectories and other files. Then add a flag for specifying another directory name. Would <3 a PR for it, though I might not be able to merge until tonight!

**Comment by sirMackk** [comment]
> We could do it so that by default it generates a feed from .md files in the blog directory?

Yeah, that sounds great - simple and solves the issue right away. I'll get on it then.

> Would <3 a PR for it, though I might not be able to merge until tonight!

Tonight?! I'll need a few days to set up golang and get some interviews out of the way :P

**Comment by GriffinMB** [comment]
Lol. Sounds good. That'll give me time to get a config file working :)
